### ISSUE TITLE
Docs or me, someone is wrong

### ISSUE THREAD

**Issue by mpasternak** [shaib/django_18_fast_migrations#1]
Hi, thanks for a great project!

It looks that docs are a bit off. First, 'django_18_fast_migrations' is enough to add to INSTALLED_APPS.

Second, in my app .ready() method, I am monkeypatching the executor. Is there any place in your code that it is being done automatically? Looks like it is not, by default. 

I'm willing to contribute some patches, but please let me know: should we add monkey patching to our app's ready() method --- or, am I doing something wrong?

**Comment by shaib** [comment]
Hi, thanks for your kind comments!

Indeed, `'django_18_fast_migrations'` is enough to add to INSTALLED_APPS. I will update the documentation.

Second, my app includes a `migrate` command which monkeypatches the executor. It is not monkeypatched in other situations. If you monkeypatch in your `ready()`, I may be overriding that.

Do you want to share why and how you monkeypatch the executor?

**Comment by mpasternak** [comment]
Hello and thanks for a quick reply.

``pip install django_18_fast_migrations`` does **not** install ``management/commands`` directory

```
(dj18mig) MacBook-Pro-Michala:~ mpasternak$ pip install django_18_fast_migrations
Collecting django_18_fast_migrations
  Using cached django_18_fast_migrations-0.1.0-py2.py3-none-any.whl
Installing collected packages: django-18-fast-migrations
Successfully installed django-18-fast-migrations-0.1.0
(dj18mig) MacBook-Pro-Michala:~ mpasternak$ ls -las ~/envs/dj18mig/lib/python2.7/site-packages/django_18_fast_migrations
django_18_fast_migrations/                 django_18_fast_migrations-0.1.0.dist-info/ 
(dj18mig) MacBook-Pro-Michala:~ mpasternak$ ls -las ~/envs/dj18mig/lib/python2.7/site-packages/django_18_fast_migrations
total 64
 0 drwxr-xr-x   8 mpasternak  staff   272 27 maj 00:55 ./
 0 drwxr-xr-x  24 mpasternak  staff   816 27 maj 00:55 ../
 8 -rw-r--r--   1 mpasternak  staff    22 27 maj 00:55 __init__.py
 8 -rw-r--r--   1 mpasternak  staff   256 27 maj 00:55 __init__.pyc
 8 -rw-r--r--   1 mpasternak  staff   142 27 maj 00:55 apps.py
 8 -rw-r--r--   1 mpasternak  staff   625 27 maj 00:55 apps.pyc
16 -rw-r--r--   1 mpasternak  staff  6490 27 maj 00:55 migration_executor_patched.py
16 -rw-r--r--   1 mpasternak  staff  7488 27 maj 00:55 migration_executor_patched.pyc
(dj18mig) MacBook-Pro-Michala:~ mpasternak$ 
```

Perhaps this is why I need to ``monkeypatch()`` in ``ready()``

By the way, any particular reason why doesn't ``django_18_fast_migrations`` use ``monkeypatch()`` in it's own ``apps.py`` ?

**Comment by shaib** [comment]
I believe both issues are fixed in version 0.1.1.

I chose to monkeypatch in the `migrate` command, rather than `ready()`, in order to minimize the effect of the monkeypatch. This way, there's less to test, and less to worry about.
